SYSTEMS.md

```diff
@@ -71,6 +71,25 @@ reframe -c examples/sombrero -r --performance-report --system csd3-skylake:compu
 where `<ACCOUNT>` is the project you want to charge.
 You can see the account balance of your projects with the `mybalance` command.
 
+## DIaL2
+
+### Queue options
+
+When submitting jobs to compute nodes, you need to specify the job queue, with the `--account` option to the scheduler.
+To do this, when you run a benchmark you can use the `-J`/`--job-option` flag to `reframe` to specify the account, for example:
+
+```
+reframe -c examples/sombrero -r --performance-report --system dial2:compute-node -J'--accout=<ACCOUNT>'
+```
+
+where `<ACCOUNT>` is the project you want to charge.
+
+This platform possesses the pbs/torque scheduler. In order to obtain and use the correct header option (`#PBS -l nodes=?:ppn=?`)
+the following additional flags are required (tested with sphng, trove, trove-pdsyev mpi/openmp hybrid apps):
+
+* `-S executable='-perhost $NUM_TASKS_PER_NODE <NAME-OF-EXECUTABLE>'`
```

Comment:
Can this be achieved automatically with a custom launcher like https://github.com/ukri-excalibur/excalibur-tests/blob/615beef981c8bc4a1a243c5adedc365f4693929d/benchmarks/reframe_config.py#L5-L10

Reply:
Do you know if there is a way to differentiate for intel and openMPI?

Reply:
Ah, that's a bit annoying, for the linked COSMA7 partition I have two separate partitions, one with Intel MPI in the spack environment, and the other one with OpenMPI.  I wasn't particularly happy about duplicating the partition, but when launcher-specific configuration is needed that might be the easiest thing to do, for some definition of "easy".

Reply:
@giordano So just to be clear, would you like me to make two separate partitions for Dial2.5? or keep the code as is?

Reply:
Actually I had another thought. This will still require manual work on dial2.5 because users will need to overwrite the running partition anyway. I'd be inclined to leave this as is.

Reply:
Uhm, ok.

Reply:
sorry I appreciate it's not ideal but we will be getting rid of dial2.5 soon (possible this year) so it doesn't feel like the best use of time.